Source/WebCore/page/LocalFrame.cpp

```diff
@@ -134,23 +134,23 @@ DEFINE_DEBUG_ONLY_GLOBAL(WTF::RefCountedLeakCounter, frameCounter, ("Frame"));
 
 static inline float parentPageZoomFactor(LocalFrame* frame)
 {
-    LocalFrame* parent = dynamicDowncast<LocalFrame>(frame->tree().parent());
+    RefPtr parent = dynamicDowncast<LocalFrame>(frame->tree().parent());
     if (!parent)
         return 1;
     return parent->pageZoomFactor();
```

Comment:
This looks like a trivial inline function, no? So why are we doing smart pointer adoption here? Please follow the guidelines.

Reply:
Sorry, indeed I did not check for trivial inline functions. Fixed this one now and went over all other changes to check for this.